app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class PagesController < ApplicationController
+  def show; end
```

Comment:
There is no 'show' action defined in routes.
And you don't need add action which doesn't contain any code.
Please remove this.